compiler/src/dotty/tools/dotc/util/Stats.scala

```diff
@@ -21,7 +21,7 @@ import collection.mutable
   }
 
   @inline
-  def record(fn: String, n: Int = 1) =
+  def record(fn: => String, n: => Int = 1) =
```

Comment:
One can do even better by using two overloaded versions of `record` and no default argument.

Reply:
I don't see how, could you give more details?